sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2264,6 +2264,15 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val SCRIPT_TRANSFORMATION_EXIT_TIMEOUT =
+    buildConf("spark.sql.scriptTransformation.exitTimeoutInSeconds")
+    .internal()
```

Comment:
nit: 2 space indentation, starting from this line.

Reply:
Sure, fixed